damien/models/department.py

```diff
@@ -104,13 +105,15 @@ def find_by_name(cls, dept_name):
         return query.first()
 
     @classmethod
-    def all_enrolled(cls):
-        query = cls.query.filter_by(is_enrolled=True)
+    def all_enrolled(cls, load_contacts=False):
+        query = cls.query.filter_by(is_enrolled=True).order_by(cls.dept_name)
+        if load_contacts:
+            query = query.options(joinedload(cls.members).joinedload('user'))
         return query.all()
 
     def catalog_listings_map(self):
         listings_map = {}
-        for listing in self.catalog_listings:
+        for listing in sorted(self.catalog_listings, key=lambda cl: [cl.subject_area, extract_int(cl.catalog_id)]):
```

Comment:
We eager-load department contacts for performance reasons, which in turn caused some mysterious changes to default catalog listing ordering. To keep tests happy, this PR includes a side trip to explicitly order the catalog listings.